### PR TITLE
Feat(git):Get pr found using WtId

### PR DESCRIPTION
**Pull request by Aditya3034**
### Functionality:
Get pr found using WtId.
### Solution:
Changes in git & task-details component and also the pr template in github
### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
1.Create a pull request in github with your Worktez Id.
2.Go to worktez.com 
3. Open task details of task mentioned with Worktez Id.
4. Click on pull request icon
5. You can see your pr found in the list.

### CODE REVIEW

File: src/app/body/task-details/git/git.component.css [hunk 1]
```diff
@@ -1,33 +1,40 @@
...
+.list-group-item {
     background-color: var(--secondary-bg);
     color: var(--text);
     cursor: pointer;
 }
+
 .modal-content {
     background-color: var(--secondary-bg);
-    border-radius: 8px;
+    border-radius: 0.3rem;
```
Comment (viv9k) [inline src/app/body/task-details/git/git.component.css:9]: border radius is always 5px

File: src/app/body/task-details/git/git.component.html [hunk 2]
```diff
@@ -19,52 +19,81 @@
...
                 <div class="row px-4">
                     <ul class="navbar-nav mr-auto">
-                            <h4 class="active nav-link head bold" id="memberModalLabel">Link Github PR</h4>
+                        <h4 class="active nav-link head bold" id="memberModalLabel">Link Github PR</h4>
                     </ul>
                     <ul class="nav navbar-nav navbar-right">
-                            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                                <span aria-hidden="true" class="material-icons text" (click)="close()">close</span>
-                            </button>
+                        <span class="prBtn btn" (click)="close()">
```
Comment (viv9k) [inline src/app/body/task-details/git/git.component.html:25]: add class align-middle to span tag

File: src/app/body/task-details/git/git.component.css [hunk 3]
```diff
@@ -59,11 +67,28 @@
...
+.prBtn:hover,
+#button:hover {
+    background-color: var(--dropdown-hover);
+    color: var(--button-text-hover);
+    border: none;
+}
+
+.close {
+    color: var(--text);
+    justify-content: center;
```
Comment (viv9k) [inline src/app/body/task-details/git/git.component.css:93]: justify center can be removed

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the following changes

**Comment by Aditya3034** [comment]
Did the required changes.

File: src/app/body/task-details/task-details.component.css [hunk 4]
```diff
@@ -152,4 +152,33 @@ a.nav-link{
     #headerLabel {
         font-size: 10px;
     }
+}
+
+.prBtn{
+    width: max-content;
+    border-radius: 6px;
```
Comment (viv9k) [inline src/app/body/task-details/task-details.component.css:159]: border-radius must be 5px

**Review by viv9k — CHANGES_REQUESTED** [review]
please do the required fix

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**